ic-task-scheduler/src/scheduler.rs

```diff
@@ -548,7 +571,7 @@ mod test {
 
                     for i in 0..10 {
                         // Should not run the task because the execution timestamp is in the future
-                        scheduler.run_with_timestamp(timestamp + i).unwrap();
+                        scheduler.run_with_timestamp((), timestamp + i).unwrap();
```

Comment:
Maybe, but I'm not sure you can use Self::Ctx::default() instead of (), but maybe not

Reply:
The Ctx generic is removed, so left it as is.